pages/common/rg.md

```diff
@@ -0,0 +1,23 @@
+# ripgrep
+
+> A fast command-line search tool.
+
+- Recursively search the current directory for a regex pattern:
+
+`rg {{pattern}}`
+
+- Search for pattern including all .gitignored and hidden files:
+
+`rg -uu {{pattern}}`
+
+- Search for a pattern only in a certain filetype (e.g., html, css, etc.):
+
+`rg -t {{filetype}} {{pattern}}`
+
+- Search for a pattern only in a subset of directories:
+
+`rg {{pattern}} {{set_of_subdirs}}`
```

Comment:
@kalebo what is the expected format for set_of_subdirs? a space-separated list of paths, I presume?